asistente_ladm_col/gui/change_detection/changes_all_parcels_panel.py

```diff
@@ -71,67 +72,90 @@ def __init__(self, parent, utils, filter_parcels=dict()):
-                    
-                item = QTableWidgetItem(status_display)
-                item.setData(Qt.UserRole, {base_db.names.T_ID_F: parcel_attrs[base_db.names.T_ID_F], 'inverse': inverse})
-                self.tbl_changes_all_parcels.setItem(row, 1, item)
-                color = STATUS_COLORS[status]
-                self.tbl_changes_all_parcels.item(row, 1).setBackground(color)
-
-                row += 1
-
-        self.tbl_changes_all_parcels.setSortingEnabled(True)
+        filter_parcels = None
+        ids_plots_supplies = list()
+        ids_plots_collected = list()
+        for type in types_change_detection:
+            filter_parcels = dict_parcels[type]
 
-        # Zoom and flash features
-        if filter_parcels:
-            plot_layer = None
             if filter_parcels[SOURCE_DB] == COLLECTED_DB_SOURCE:
-                plot_layer = self.utils._layers[self.utils._db.names.OP_PLOT_T][LAYER]
+                inverse = False
             else:
-                plot_layer = self.utils._supplies_layers[self.utils._supplies_db.names.OP_PLOT_T][LAYER]
+                inverse = True  # Take the supplies db as base db
+
+            base_db = self.utils._supplies_db if inverse else self.utils._db
+            self.compared_parcels_data = self.utils.get_compared_parcels_data(inverse)
+
+            for parcel_number, parcel_attrs in self.compared_parcels_data.items():
+                if filter_parcels and parcel_number in filter_parcels[DICT_KEY_PARCEL_T_PARCEL_NUMBER_F]:
+                    item = QTableWidgetItem(parcel_number) if parcel_number else QTableWidgetItem(QgsApplication.nullRepresentation())
+                    item.setData(Qt.UserRole, {base_db.names.T_ID_F: parcel_attrs[base_db.names.T_ID_F], 'inverse': inverse})
+                    self.tbl_changes_all_parcels.setItem(row, 0, item)
+
+                    status = parcel_attrs[PARCEL_STATUS]
+                    status_display = parcel_attrs[PARCEL_STATUS_DISPLAY]
+                    if filter_parcels:
+                        # If we are on the supplies DB, "new" parcels are "missing" parcels from the collected db perspective
+                        if filter_parcels[SOURCE_DB] == SUPPLIES_DB_SOURCE and parcel_attrs[PARCEL_STATUS_DISPLAY] == CHANGE_DETECTION_NEW_PARCEL:
+                            status_display = CHANGE_DETECTION_MISSING_PARCEL
+                            status = CHANGE_DETECTION_MISSING_PARCEL
+
+                    item = QTableWidgetItem(status_display)
+                    item.setData(Qt.UserRole, {base_db.names.T_ID_F: parcel_attrs[base_db.names.T_ID_F], 'inverse': inverse})
+                    self.tbl_changes_all_parcels.setItem(row, 1, item)
+                    color = STATUS_COLORS[status]
+                    self.tbl_changes_all_parcels.item(row, 1).setBackground(color)
+
+                    row += 1
+
+            self.tbl_changes_all_parcels.setSortingEnabled(True)
+
+
+            if filter_parcels:
```

Comment:
TODO: List of parcel ids and after the for, call select and zoom to compound bounding box once.

Reply:
Solved in https://github.com/AgenciaImplementacion/Asistente-LADM_COL/pull/284/commits/3584adad6dd43308c11613de6dc0d7469f98b623